xarray/core/variable.py

```diff
@@ -1261,7 +1270,7 @@ def equals(self, other, equiv=duck_array_ops.array_equiv):
             return (self.dims == other.dims and
                     (self._data is other._data or
                      equiv(self.data, other.data)))
-        except (TypeError, AttributeError):
+        except (TypeError, AttributeError) as e:
```

Comment:
it doesn't look like `e` is ever used.

Reply:
Good catch, this was introduced in debugging